packages/d3fc-series/src/webgl/bar.js

```diff
@@ -10,15 +10,15 @@ export default () => {
 
     let equals = (previousData, data) => false;
     let previousData = [];
+    let filteredData = [];
```

Comment:
I'm not sure this is the right scope, can this be moved into the function below?

Reply:
If this is in the function below and the data is equal (we aren't reprocessing) then we will using an empty array for the `decorate` function and the `draw` call.

If we don't store the filtered data between renders then I think we will have to call `defined` every render

Reply:
Deferred until https://github.com/d3fc/d3fc/issues/1412